transbank/webpay/webpayplus/transaction.py

```diff
@@ -0,0 +1,60 @@
+import json
+
+from transbank.webpay.webpayplus.transaction_create_response import TransactionCreateResponse
+from transbank.webpay.webpayplus.webpayplus import *
+
+
+class Transaction:
+    CREATE_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions'
+    COMMIT_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions'
+    REFUND_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions/{0}/refund'
+    GET_TRANSACTION_STATUS_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions/{0}'
+
+    def __init__(self):
+        pass
+
+    @classmethod
+    def create(cls, buy_order, session_id, amount,
+               return_url, options=None):
+        commerce_code = None
+        api_key = None
+        base_url = None
+        if options is None:
+            commerce_code = WebpayPlus.commerce_code()
+            api_key = WebpayPlus.api_key()
+            base_url = WebpayPlus.integration_type_url()
+        else:
+            commerce_code = options.commerce_code
```

Comment:
What happen if I want to overwrite only the `api_key`?? if I set only this property on the Options object then I will get an error because the `commerce_code` will get a `None` value and the `base_url` could get an unspected value.